src/Migration.php

```diff
@@ -52,4 +50,15 @@ public function run_migrations() {
 	public static function wc_blocks_update_710_blockified_product_grid_block() {
 		update_option( Options::WC_BLOCK_USE_BLOCKIFIED_PRODUCT_GRID_BLOCK_AS_TEMPLATE, wc_bool_to_string( false ) );
 	}
+
+	/**
+	 * Sets the $db_upgrades array value.
+	 *
+	 * @param array $db_upgrades Array of migrations to run.
+	 *
+	 * @return void
+	 */
+	public function set_db_upgrades( array $db_upgrades ) {
```

Comment:
Can you explain why did you add this function?

Reply:
I removed the reflection class because I was getting weird issues (a mix of PHP incompatibility and ci cache after all) and replaced that with a setter to keep code simple, and forgot to change it back.

Nice catch, I'm reverting that 👍🏼